src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/UserGroupController.kt

```diff
@@ -2,19 +2,34 @@ package org.vaccineimpact.orderlyweb.controllers.web
 
 import org.vaccineimpact.orderlyweb.ActionContext
 import org.vaccineimpact.orderlyweb.controllers.Controller
+import org.vaccineimpact.orderlyweb.db.AppConfig
 import org.vaccineimpact.orderlyweb.db.AuthorizationRepository
 import org.vaccineimpact.orderlyweb.db.OrderlyAuthorizationRepository
 import org.vaccineimpact.orderlyweb.db.OrderlyUserRepository
 import org.vaccineimpact.orderlyweb.errors.MissingParameterError
 import org.vaccineimpact.orderlyweb.models.Scope
 import org.vaccineimpact.orderlyweb.models.permissions.AssociatePermission
 import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
+import org.vaccineimpact.orderlyweb.viewmodels.AdminViewModel
+import org.vaccineimpact.orderlyweb.viewmodels.ReportVersionPageViewModel
 
 class UserGroupController(context: ActionContext,
                           private val authRepo: AuthorizationRepository) : Controller(context)
 {
     constructor(context: ActionContext) : this(context, OrderlyAuthorizationRepository())
 
+    @Template("admin.ftl")
+    fun admin(): AdminViewModel
```

Comment:
Probably not where this method should live, but starting a new AdminController felt a bit wrong too - what distinguishes the AdminController from the UserGroupController (which does manage permissions at the moment) and the RoleController? Need a bit of a rethink about sensible structure here.

Reply:
I see what you mean, particularly since this route is just going to return an empty page, with the actual data to be retried asynchronously..